android/src/main/java/ee/forgr/plugin/capacitor_purchases/CapacitorPurchasesPlugin.java

```diff
@@ -38,8 +38,9 @@ public class CapacitorPurchasesPlugin extends Plugin {
     public void setup(PluginCall call) {
         String apiKey = call.getString("apiKey");
         String appUserID = call.getString("appUserID");
+        String observerMode = call.getBoolean("observerMode") != null ? call.getBoolean("observerMode") : false;
         PlatformInfo platformInfo = new PlatformInfo(PLATFORM_NAME, PLUGIN_VERSION);
-        CommonKt.configure(this.bridge.getActivity(), apiKey, appUserID, true, platformInfo);
+        CommonKt.configure(this.bridge.getActivity(), apiKey, appUserID, observerMode, platformInfo);
```

Comment:
I believe #10 would already be solved by just passing in `false` here, to match iOS settings. But having the option is useful.

Reply:
I think 
`String observerMode = call.getBoolean("observerMode") != null ? call.getBoolean("observerMode") : false;`

should be `Boolean observerMode = call.getBoolean("observerMode") != null ? call.getBoolean("observerMode") : false;`

Reply:
LOL yeah, my bad! Updating. Thanks!

Reply:
updated

Reply:
@aboedo you can pass default value even better
```
String observerMode = call.getBoolean("observerMode", false);
```

Reply:
thanks! that is much nicer 💪